crates/holochain/src/lib.rs

```diff
@@ -2,6 +2,7 @@
 
 // We have a lot of usages of type aliases to `&String`, which clippy objects to.
 #![allow(clippy::ptr_arg)]
+#![recursion_limit = "128"]
```

Comment:
Isn't 128 the default? Should we remove this line?

Reply:
Oops forgot to before merging, but feel free to remove it if I don't in another branch